lib/DMV_setup.rb

```diff
@@ -0,0 +1,50 @@
+class DMVSetup
```

Comment:
Remember that our convention for file names is snake case. So you might have wanted to call this class `DmvSetup` so the file could be called `dmv_setup.rb`